manim/mobject/types/vectorized_mobject.py

```diff
@@ -876,6 +876,14 @@ def __init__(self, *vmobjects, **kwargs):
         VMobject.__init__(self, **kwargs)
         self.add(*vmobjects)
 
+    def __repr__(self):
+        return (
+            __class__.__name__
```

Comment:
shouldn't this be `self.__class__`?

Reply:
I just checked. Either of them gives the same output. I guess it's better to go with `self` then.